### ISSUE TITLE
Remove contents of `notebooks` from git history.

### ISSUE THREAD

**Issue by Jordan-Dennis** [LouisDesdoigts/dLux#165]
Hi all,
The `git` repository currently occupies a quarter of a gigabyte, which seems like it is far too large. After running `du -h .git` I find that the majority of the large files are contained within the old (now deleted and ignored) `notebooks` directory. I would like to permanently delete these from the `git` repository to reduce its size. 

I will format this as a separate pull request to #146 in the coming hours. 

Regards

Jordan

**Comment by Jordan-Dennis** [comment]
![image](https://user-images.githubusercontent.com/93350879/210020848-44eb7646-f646-4058-94e8-d3a2d4de6075.png)
Is the actual image.

**Comment by Jordan-Dennis** [comment]
OK, so this is way harder than I thought it was. I've just spent around an hour trying and I think that I will leave it alone for now.

**Comment by LouisDesdoigts** [comment]
After 4 years, I have finally reduced the repo size by an order of magnitude by nuking old notebooks and files that shouldn't have been in the repo 🙏

**Comment by maxecharles** [comment]
🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏

**Comment by maxecharles** [comment]
> OK, so this is way harder than I thought it was.

These words echoed throughout the years

**Comment by LouisDesdoigts** [comment]
The ~general~ Pope scientific process:

1. "it will just work"
2. OK, so this is way harder than I thought it was.
3. Suffering
4. It finally works
5. "See, it just worked!"

**Comment by maxecharles** [comment]
> The ~general~ Pope scientific process:
> 
> 1. "it will just work"
> 2. OK, so this is way harder than I thought it was.
> 3. Suffering
> 4. It finally works
> 5. "See, it just worked!"

Currently in step 3. for the amigo retraining

**Comment by LouisDesdoigts** [comment]
Yes unfortunately that step can last for anywhere between 20-98% of a project
